### PR TITLE
Added button for call for papers and countdown

### PR DESCRIPTION
**Pull request by Archel**

### CODE REVIEW

File: assets/js/theme.js [hunk 1]
```diff
@@ -211,7 +211,7 @@ var theme = function () {
         },
         // CountDown
         initCountDown: function () {
-            var austDay = new Date(2017,9,7);
+            var austDay = new Date(2018,8,29);
```
Comment (JavierCane) [inline assets/js/theme.js]: Shouldn't it be `2018,8,29`? (the conference is taking place on _September_ 29th)

Comment (Archel) [reply assets/js/theme.js]: In Javascript the months are between 0 and 11.

Comment (Archel) [reply assets/js/theme.js]: Is the change that I made, isn't?

Comment (nubeiro) [reply assets/js/theme.js]: so the 8th month is September? :thinking: I thought that was August :-D

Comment (nubeiro) [reply assets/js/theme.js]: lol, it is, my bad :/ this just show how much Js I write in a daily basis

Comment (JavierCane) [reply assets/js/theme.js]: My bad @archel. As @nubeiro I also haven't taken into account zero based JS months.

**Comment by JavierCane** [comment]
Merged! 🙂🙌 @archel

**Pull request merged**